rs/sns/root/canister/canister.rs

```diff
@@ -356,16 +359,6 @@ fn assert_eq_governance_canister_id(id: PrincipalId) {
     });
 }
 
-#[heartbeat]
-async fn heartbeat() {
-    // Asynchronous method called for the canister_heartbeat that injects
-    // dependencies to run_periodic_tasks.
-    let now = CanisterEnvironment {}.now();
-    let ledger_client = create_ledger_client();
-
-    SnsRootCanister::heartbeat(&STATE, &ledger_client, now).await
```

Comment:
Should we also remove the heartbeat method from SnsRootCanister?

Reply:
Done.